app-text/zathura/zathura-0.5.12.ebuild

```diff
@@ -0,0 +1,68 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit flag-o-matic meson xdg
+
+DESCRIPTION="Highly customizable & functional document viewer"
+HOMEPAGE="https://pwmt.org/projects/zathura/"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/pwmt/zathura.git"
+else
+	SRC_URI="https://github.com/pwmt/zathura/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~riscv ~x86 ~amd64-linux ~x86-linux"
+fi
+
+LICENSE="ZLIB"
+SLOT="0/6.7" # plugin versions api.abi (see meson.build)
+IUSE="+man seccomp synctex test wayland X"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="
+	test? ( X )
+	|| ( wayland X )
+"
+
+RDEPEND="
+	dev-libs/json-glib
+	x11-libs/cairo
+	x11-libs/pango
+	dev-db/sqlite:3
+	>=dev-libs/girara-0.4.5:=[X?]
+	>=dev-libs/glib-2.72:2
```

Comment:
Bump to 2.76 (0.5.12) : https://github.com/pwmt/zathura/commit/4ae0e8c084b6703c42fa149cf6685c365d0ef95c